### PR TITLE
feat(cli): add CI

### LINKED ISSUE #8: Add CI on CLI

### PR DESCRIPTION
**Pull request by angela-hubner**

### CODE REVIEW

File: .github/workflows/c3pm.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: c3pm CI
+
+on:
+  pull_request:
+    paths:
+      - 'cli/**'
```
Comment (Codelax) [inline .github/workflows/c3pm.yml]: This was the path used in old monorepo tree

File: .github/workflows/c3pm.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: c3pm CI
+
+on:
```
Comment (Codelax) [inline .github/workflows/c3pm.yml:3]: are `on` tags useful with current values ?

Comment (gabrielcolson) [reply .github/workflows/c3pm.yml:3]: I think we can remove it indeed

**Review by gabrielcolson — APPROVED** [review]

File: .github/workflows/c3pm.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  pull_request:
+    paths:
+      - .github/workflows/c3pm.yml
```
Comment (gabrielcolson) [inline .github/workflows/c3pm.yml]: This must be deleted

**Review by gabrielcolson — CHANGES_REQUESTED** [review]

**Review by gabrielcolson — APPROVED** [review]

**Review by Codelax — APPROVED** [review]

**Pull request merged**